packages/host/app/components/operator-mode/submode-layout.gts

```diff
@@ -333,16 +350,18 @@ export default class SubmodeLayout extends Component<Signature> {
               @displayName={{this.matrixService.profile.displayName}}
             />
           </button>
-          <SearchSheet
-            @mode={{this.searchSheetMode}}
-            @onSetup={{this.setupSearch}}
-            @onBlur={{this.closeSearchSheet}}
-            @onCancel={{this.closeSearchSheet}}
-            @onFocus={{this.openSearchSheetToPrompt}}
-            @onSearch={{this.expandSearchToShowResults}}
-            @onCardSelect={{this.handleCardSelectFromSearch}}
-            @onInputInsertion={{this.storeSearchElement}}
-          />
+          {{#if @onCardSelectFromSearch}}
+            <SearchSheet
+              @mode={{this.searchSheetMode}}
+              @onSetup={{this.setupSearch}}
+              @onBlur={{this.closeSearchSheet}}
+              @onCancel={{this.closeSearchSheet}}
+              @onFocus={{this.openSearchSheetToPrompt}}
+              @onSearch={{this.expandSearchToShowResults}}
+              @onCardSelect={{this.handleCardSelectFromSearch}}
+              @onInputInsertion={{this.storeSearchElement}}
+            />
+          {{/if}}
```

Comment:
Since the design doesn’t show a search feature, this argument is now optional, providing it means you want search.